datasets/clickbait_news_bg/clickbait_news_bg.py

```diff
@@ -0,0 +1,118 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Dataset with clickbait and fake news in Bulgarian. """
+
+from __future__ import absolute_import, division, print_function
+
+import pandas as pd
+
+import datasets
+
+
+_CITATION = """\
+@InProceedings{clickbait_news_bg,
+title = {Dataset with clickbait and fake news in Bulgarian. Introduced for the Hack the Fake News 2017.},
+authors={Data Science Society},
+year={2017},
+url={https://gitlab.com/datasciencesociety/case_fake_news/}
+}
+"""
+
+# TODO: Add description of the dataset here
+# You can copy an official description
+_DESCRIPTION = """\
+Dataset with clickbait and fake news in Bulgarian. Introduced for the Hack the Fake News 2017.
+"""
+
+# TODO: Add a link to an official homepage for the dataset here
+_HOMEPAGE = "https://gitlab.com/datasciencesociety/case_fake_news/"
+
+# TODO: Add the licence for the dataset here if you can find it
+_LICENSE = ""
+
+# TODO: Add link to the official dataset URLs here
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URLs = {
+    "default_train": "https://gitlab.com/datasciencesociety/case_fake_news/-/raw/master/data/FN_Training_Set.xlsx",
+    "default_validation": "https://gitlab.com/datasciencesociety/case_fake_news/-/raw/master/data/FN_Validation_Set.xlsx",
+}
+
+
+# TODO: Name of the dataset usually match the script name with CamelCase instead of snake_case
+class ClickbaitNewsBGDataset(datasets.GeneratorBasedBuilder):
+    VERSION = datasets.Version("1.1.0")
+    DEFAULT_CONFIG_NAME = "default"
+
+    def _info(self):
+        if self.config.name == "default":
+            features = datasets.Features(
+                {
+                    "fake_news_score": datasets.Value("int8"),
+                    "click_bait_score": datasets.Value("int8"),
```

Comment:
What are the values this feature can take ?
It feels like `click_bait_score` and `fake_news_score` could be `ClassLabel` features

Reply:
It looks like the labels take only the value 1 (negative prediction) and 3 (positive prediction).
You can use ClassLabel this way for example:
```
"click_bait_score": ClassLabel(names=["normal", "click bait"])
"fake_news_score": ClassLabel(names=["legitimate", "fake"])
```
(label names come from [here](https://gitlab.com/datasciencesociety/case_fake_news/-/blob/master/case_study/Hackaton_Task.pdf))

You will also have to convert all the "1" and the "3" accordingly when you yield the examples in _generate_examples:
- for click_bait_score:
  - 1 -> "normal"
  - 3 -> "click bait"
- for fake_news_score:
  - 1 -> "legitimate"
  - 3 -> "fake"